src/ToDoList/ToDoList.tsx

```diff
@@ -1,35 +1,56 @@
 import React from "react";
 import ToDoItem from "../ToDoItem/ToDoItem";
 import { TodoItem } from "../types";
-import Box from "@mui/material/Box";
+import { Container, Box } from "@mui/material";
+import { Droppable } from "react-beautiful-dnd";
+import { DragDropContext } from "react-beautiful-dnd";
+import { resourceUsage } from "process";
+import type { DroppableProvided } from "react-beautiful-dnd";
 
 interface ToDoListProps {
   items: TodoItem[];
   onDelete: (id: number) => void;
   onToggle: (id: number) => void;
+  onDragEnd: (result: any) => void;
 }
 
-const ToDoList: React.FC<ToDoListProps> = ({ items, onDelete, onToggle }) => {
+const ToDoList: React.FC<ToDoListProps> = ({
+  items,
+  onDelete,
+  onToggle,
+  onDragEnd,
+}) => {
   return (
-    <Box
-      sx={{
-        display: "flex",
-        flexDirection: "column",
-        alignItems: "center",
-        rowGap: 2,
-        width: "100%",
-      }}
-    >
-      {items.map((item) => (
-        <ToDoItem
-          key={item.id}
-          description={item.description}
-          completed={item.completed}
-          onToggle={() => onToggle(item.id)}
-          onDelete={() => onDelete(item.id)}
-        />
-      ))}
-    </Box>
+    <DragDropContext onDragEnd={onDragEnd}>
+      <Droppable droppableId={"todo-tasks"}>
+        {(provided: any, snapshot: any) => (
```

Comment:
Same here. Do not use `any`. Btw, you are not using `snapshot` variable here, you should remove it.